charts/firefly/templates/evmconnect/secret.yaml

```diff
@@ -0,0 +1,80 @@
+{{/*
+  Copyright © 2022 Kaleido, Inc.
+
+  SPDX-License-Identifier: Apache-2.0
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://swww.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/}}
+
+{{- if .Values.evmconnect.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "firefly.fullname" . }}-evmconnect-config
+  labels:
+    {{- include "firefly.evmconnectLabels" . | nindent 4 }}
+stringData:
+  config.yaml: |-
+    {{- with .Values.evmconnect.config.confirmations }}
+    confirmations:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.eventstreams }}
+    eventstreams:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.webhooks }}
+    webhooks:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.policyLoop }}
+    policyloop:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- with .Values.evmconnect.config.transactions }}
+    transactions:
+      {{- toYaml . | nindent 6 }}
+    {{- end }}
+
+    {{- tpl .Values.evmconnect.config.policyEngine . | nindent 4 }}
+
+    log:
+      level: {{ .Values.evmconnect.config.log.level | quote }}
+      json:
+        enabled: {{ .Values.evmconnect.config.log.jsonEnabled }}
+
+    persistence:
+      type: leveldb
+      leveldb:
+        maxHandles: {{ .Values.evmconnect.config.leveldb.maxHandles | int }}
+        path: /var/run/leveldb
+        syncWrites: {{ .Values.evmconnect.config.leveldb.syncWrites }}
+
+    connector:
+      {{- tpl .Values.evmconnect.config.connector . | nindent 6 }}
```

Comment:
A connector (i.e. the blockchain) is required, and so this template defaults to simply setting the JSONRPC URL of the node to use but offers customizing the connector config if necessary.